packages/core/src/schema/models/evaluationResults.ts

```diff
@@ -21,22 +21,38 @@ export const evaluationResults = latitudeSchema.table(
     evaluationId: bigint('evaluation_id', { mode: 'number' })
       .notNull()
       .references(() => evaluations.id, { onDelete: 'cascade' }),
+    // TODO: remove
     documentLogId: bigint('document_log_id', { mode: 'number' })
```

Comment:
Are we losing something by not having the reference to the log? I guess we always can get it through provider log

Reply:
n